src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,31 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
 }
 
+type UpdateHomepageType =
+  | "editor"
+  | "dropdownelem"
+  | "highlight"
+  | "announcement"
+  | `textcardcard-${number}`
+
+const isUpdateHomepageType = (value: any): value is UpdateHomepageType => {
+  if (typeof value === "string" && value.startsWith("textcardcard-")) {
+    const valArr = value.split("-")
+    const number = valArr[1]
+    return valArr.length === 2 && !Number.isNaN(Number(number))
+  }
+  if (typeof value === "string") {
```

Comment:
nit: can be an else if

Reply:
My linter is removing the else ifs here actually, since the if statement returns haha